### PR TITLE
Add `Hvendor` and `Hname` as attributes of `accept`, `require` and `deny` elements

### PR DESCRIPTION
**Pull request by silviooliva**
Commit related to issue #182. This allows part specific filtering.

### CODE REVIEW

**Review by jkrech — CHANGES_REQUESTED** [review]
Thanks a lot for the PR.
Please see my comment. Are you happy to change to Hvendor to xs:string. This is similar to Bvendor where we don't have an fixed list with vendorID numbers.

**Comment by silviooliva** [comment]
> Thanks a lot for the PR. Please see my comment. Are you happy to change to Hvendor to xs:string. This is similar to Bvendor where we don't have an fixed list with vendorID numbers.

@jkrech, 
I've used the `DeviceVendorEnum` since the the `Hvendor` has been defined as DeviceVendorEnum type (see the [`part`](https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/pdsc_parts_pg.html#element_part) element description).
Changing the `Hvendor` here to _xs:string_, maybe we should also change the `Hvendor` attribute definition in the `part` element (no problem on my side to fix it as well). Or do you think we could keep the two definitions disaligned?

**Comment by jkrech** [comment]
@silviooliva, good catch. I do agree that we should also make it xs:string for the part element:
https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/pdsc_parts_pg.html#element_part
Could you please do so?

**Comment by silviooliva** [comment]
@jkrech,
after further thinking, I'm convinced too it is better, for the same reason you exposed [here](https://github.com/Open-CMSIS-Pack/Open-CMSIS-Pack-Spec/pull/228#discussion_r1164199307), to use _xs:string_ for the Hvendor. 
I'm going to fix it.

File: schema/PACK.xsd [hunk 1]
```diff
@@ -1286,6 +1288,8 @@
     <xs:attribute name="Bvendor" type="xs:string" />
     <xs:attribute name="Bname" type="xs:string" />
     <xs:attribute name="Brevision" type="xs:string" />
+    <xs:attribute name="Hvendor" type="DeviceVendorEnum" />
```
[the remark this answers is not in the document]
Comment (silviooliva) [reply schema/PACK.xsd]: Fixed.

**Review by jkrech — APPROVED** [review]
Great. Thanks for making updates in all affected locations/files. LGTM.

**Pull request merged**